### PR TITLE
Search: allow the Search team to approve changes to the package

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* The Search package is the work of the Search team. They're familiar with its structure and codebase, they should be able to approve changes just like Jetpack Approvers currently can.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Not much to test in this PR.

### CODE REVIEW

**Review by anomiex — COMMENTED** [review]
This won't work as written since the "Jetpack and packages" block will still require a Crew review. See on #22461 we have it looking for a reviewer at both [here](https://github.com/Automattic/jetpack/runs/4926796468?check_suite_focus=true#step:3:23) and [here](https://github.com/Automattic/jetpack/runs/4926796468?check_suite_focus=true#step:3:37)

Adding a pattern `'!projects/packages/search/**'` in there (around line 24-ish) to exclude the search package from that block should fix it.

**Comment by jeherve** [comment]
thanks for catching that! Pushed your suggestion in 78bd5309972b9a9b0391a9cad48dab56e99f9c8f

File: .github/files/required-review.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@
   paths:
    - 'projects/packages/**'
    - 'projects/plugins/jetpack/**'
+   - '!projects/packages/search/**'
```
Comment (anomiex) [inline .github/files/required-review.yaml:25]: I wonder if it's worth a comment pointing out why we have this line here.
```suggestion
   # Exclude packages managed by other teams, which are covered by other blocks below.
   - '!projects/packages/search/**'
```

**Review by anomiex — APPROVED** [review]
Looks good, but one (optional) suggestion inline

**Review by kraftbj — APPROVED** [review]

**Pull request merged**